app/api/v1/endpoints/unified_chat.py

```
@@ -0,0 +1,710 @@
+    except Exception as e:
+        logger.error("Failed to get user sessions", error=str(e), user_id=str(current_user.id))
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail="Failed to retrieve chat sessions"
+        )
+
+
+# Create New Session
+@router.post("/session/new")
+async def create_new_session(
+    current_user: User = Depends(get_current_user)
```

Ruff (0.12.2)

Unused function argument: `current_user`

(ARG001)